### PR TITLE
Fixed list styles (MD004) in the Cloud Guide

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) fixes all MD004 errors in the Cloud Guide, README, and community resources page.

The MD004 rule requires consistent use of unordered list styles. The scope of #5241 also permits—but does not require—resolving errors related to the following rules:

- MD005: List indentation
- MD006: Start bulleted lists at the beginning of a line
- MD007: List indentation
- MD030: Spaces after list markers

This is one of many PRs related to #5241.

### CODE REVIEW

File: guides/v2.2/cloud/release-notes/CloudReleaseNotes2.2.md [hunk 1]
```diff
@@ -57,55 +57,63 @@ For all known {{site.data.var.ee}} 2.2.0 GA issues, see [Magento Commerce 2.2.0
 
 This release contains numerous fixes and enhancements. This section includes only fixes for {{site.data.var.ece}}. For all known {{site.data.var.ee}} 2.2.0 GA issues, see [Magento Commerce 2.2.0 Release Notes]({{ site.baseurl }}/guides/v2.2/release-notes/ReleaseNotes2.2.0EE.html).
 
-<!--RFC-125-->* We removed `var/view_preprocessed` symlinking to fix an issue that was causing JavaScript minification conflicts.
+<!--RFC-125-->
```
Comment (hguthrie) [inline guides/v2.2/cloud/release-notes/CloudReleaseNotes2.2.md]: All Cloud release notes have the ticket comment inline. Is it affecting the presentation? Or is it a convention thing that you are cleaning as you see it?

File: guides/v2.2/cloud/setup/first-time-setup-import-prepare.md [hunk 2]
```diff
@@ -210,15 +226,21 @@ The following example compresses the dump so that it does not significantly inte
...
 
-        mysqldump -p -u magento magento --single-transaction --no-autocommit --quick | gzip > ~/db.sql.tgz
+   ```bash
+   mysqldump -p -u magento magento --single-transaction --no-autocommit --quick | gzip > ~/db.sql.tgz
+   ```
 
 2. Use the `rsync` command to transfer the database dump to the {{site.data.var.ece}} environment in the `var` directory of the application you are importing into:
 
-        rsync <db dump file name> <cloud SSH URL>:var/db.sql.gz
+   ```bashg
```
Comment (hguthrie) [inline guides/v2.2/cloud/setup/first-time-setup-import-prepare.md]:
```suggestion
   ```bash
```

File: guides/v2.2/cloud/setup/first-time-setup-import-prepare.md [hunk 3]
```diff
@@ -128,77 +128,93 @@ To add authentication keys using an environment variable:
...
+   git add -A && git commit -m "Add Cloud files" && git push origin <branch name>
+   ```
 
 ## Back up and transfer media files {#media}
 
 Use the command [`magento setup:backup --media`]({{ page.baseurl }}/install-gde/install/cli/install-cli-backup.html) to back up media files:
 
-1.  Get the  integration system's [SSH URL]({{ page.baseurl }}/cloud/setup/first-time-setup-import-first-steps.html#ssh).
-2.  To back up media files, enter the following command:
+1. Get the  integration system's [SSH URL]({{ page.baseurl }}/cloud/setup/first-time-setup-import-first-steps.html#ssh).
```
Comment (hguthrie) [inline guides/v2.2/cloud/setup/first-time-setup-import-prepare.md]:
```suggestion
1. Get the integration system's [SSH URL]({{ page.baseurl }}/cloud/setup/first-time-setup-import-first-steps.html#ssh).
```

File: guides/v2.2/cloud/setup/first-time-setup-import-prepare.md (hunk 2 above)
Comment (dshevtsov) [inline guides/v2.2/cloud/setup/first-time-setup-import-prepare.md]:
```suggestion
   ```bash
```

File: guides/v2.2/cloud/release-notes/CloudReleaseNotes2.2.md (hunk 1 above)
Comment (dshevtsov) [reply guides/v2.2/cloud/release-notes/CloudReleaseNotes2.2.md]: This violates [rule MD006](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md006---consider-starting-bulleted-lists-at-the-beginning-of-the-line).

File: community/resources/best-practices.md [hunk 4]
```diff
@@ -7,11 +7,11 @@ This section covers a broad scope of Community contributed best and leading prac
 
 Topics and areas include:
 
-* Project setup
-* Theme development
-* Composer use and repos
-* Local environment options
-* And much more
+*  Project setup
```
[the remark this answers is not in the document]
Comment (dshevtsov) [reply community/resources/best-practices.md]: The [rule MD004](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md004---unordered-list-style) is set to use default style, which is 'consistent'.

[the remark this answers is not in the document]
Comment (jeff-matthews) [reply community/resources/best-practices.md]: First style occurrence in the topic wins

File: guides/v2.2/cloud/release-notes/CloudReleaseNotes2.2.md (hunk 1 above)
Comment (jeff-matthews) [reply guides/v2.2/cloud/release-notes/CloudReleaseNotes2.2.md]: Yes, it violates the rule and returns errors.

File: community/resources/best-practices.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (hguthrie) [reply community/resources/best-practices.md]: Okay. Most of Cloud should be `-` so I understand what you are doing.

**Pull request merged**